static/docs/tutorials/interactive.md

```diff
@@ -0,0 +1,15 @@
+# Interactive Tutorials
+
+Learn how DVC can be used in simple ML scenarios:
+
+1. [Data Versioning](https://katacoda.com/dvc/courses/tutorials/versioning)
+   <br/> Using DVC commands to work with multiple versions of datasets and ML
```

Comment:
can we avoid using `<br/>` I think there should be way in Markdown to break a line w/o HTML, no?

Reply:
> can we avoid using <br/> I think there should be way in Markdown to break a line w/o HTML, no?

I don't think so. Please let me know if there is another way.